studio/lib/payloads/compensations.ts

```diff
@@ -8,7 +8,7 @@ export interface Benefit {
   richText: PortableTextBlock[];
```

Comment:
to be solved in a diff PR. perhaps it makes sense to cleanup our payload structure. so put static pages in one and dynamic pages in another.. I dunno. We can talk about it later